cmd/keymasterd/2fa_u2f.go

```diff
@@ -166,14 +167,14 @@ func (state *RuntimeState) u2fRegisterResponse(w http.ResponseWriter, r *http.Re
 		CreatedAt:    time.Now(),
 		CreatorAddr:  r.RemoteAddr,
 	}
+	if authUser != assumedUser {
+		newReg.Name = fmt.Sprintf("Registered by %s", authUser)
```

Comment:
dont use fmt use the state.logger, and also make the message more clear: u2f for user X registerd by user Y

Reply:
The intent is not to log a message but rather to leave the trail in the profile UI for the user to see. That way the user knows if an admin registered the U2F token, and which admin it was.